### PR TITLE
Improve/make unmute only on player click

### PR DESCRIPTION
**Pull request by sambua**

### CODE REVIEW

File: src/dynamics/video_player/player/player.js [hunk 1]
```diff
@@ -1557,28 +1557,7 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
...
-                                    once: true
-                                }));
-                            }
-                            // User interacted with player, and set player's volume level/un-mute
-                            // So we will play voice as soon as player visible for user
-                            if (!this.get("muted") && !this.get("unmuteonclick")) this.set_volume(this.get("initialoptions").volumelevel);
-                            this.set("volumeafterinteraction", true);
-                            if (this.get("forciblymuted")) this.set("forciblymuted", false);
-                        }
+                        if (strong && !this.get("volumeafterinteraction")) {}
```
Comment (ntshcalleia) [inline src/dynamics/video_player/player/player.js:1560]:
```suggestion
```

File: src/dynamics/video_player/player/player.js [hunk 2]
```diff
@@ -2556,6 +2531,7 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                                     if (opt.muted) {
                                         this.set("forciblymuted", true);
                                         this.set("autoplay-requires-muted", true);
+                                        this.set("mute", true);
```
Comment (ntshcalleia) [inline src/dynamics/video_player/player/player.js:2534]:
```suggestion
                                        this.set("muted", true);
```

**Review by ntshcalleia — CHANGES_REQUESTED** [review]
Hi @sambua, there's a minor typo in one `muted` instance, but other than that it looks good.

**Comment by sambua** [comment]
Hi @ntshcalleia yep, thanks for checking!

**Review by ntshcalleia — APPROVED** [review]

**Pull request merged**